packages/create-redwood-app/template/jest.config.js

```diff
@@ -0,0 +1,9 @@
+// This the monorepo jest config
+// Each side has their configuration setup e.g. in `web/jest.config.js`
+
+// Note that jest configs aren't merged!
```

Comment:
This copy should be improved:
- monorepo doesn't have much meaning; maybe "Redwood Project root Jest config"
- Each side, e.g. web/ and api/, has specific config files that references this root config
- add a link to Jest Project config section in redwoodjs.com/docs/testing section [TODO]